manifests/concourse-manifest/concourse-base.yml

```diff
@@ -73,21 +73,22 @@ jobs:
           databases:
             - name: atc
               role: atc
-              password: dummy-password
+              password: (( grab secrets.concourse_postgres_password ))
```

Comment:
I am split about this issue. On one hand I agree with @alext that it would be misleading to set a password that it is not being used. But on the other hand it is good to generate all the passwords.

But it is true that this line: `host   all                all                0.0.0.0/0     md5` might allow connect from other machines, so it is good to set the password.

Said that, the less effort would be leave this commit here.